dotnet/samples/Concepts/Plugins/CreatePluginFromOpenAI_AzureKeyVault.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Plugins;
 
+[Obsolete("OpenAI plugins are deprecated and will be removed in a future version.")]
 public class CreatePluginFromOpenAI_AzureKeyVault(ITestOutputHelper output) : BaseTest(output)
```

Comment:
Do we also want to add some comment to the readme that links here, e.g. (Deprecated) in brackets after the link or something similar?

Reply:
Yep, make sense. Updated.